python/rmm/allocators/torch.py

```diff
@@ -16,11 +16,14 @@
 except ImportError:
     rmm_torch_allocator = None
 else:
-    import rmm._lib.torch_allocator
+    import pathlib
 
-    _alloc_free_lib_path = rmm._lib.torch_allocator.__file__
     rmm_torch_allocator = CUDAPluggableAllocator(
-        _alloc_free_lib_path,
+        pathlib.Path(__file__).absolute().parent
+        / ".."
+        / "_lib"
+        / "libtorch_allocator.so",
```

Comment:
This has to line up with the name that cmake uses for the shared library. Ideally we would control both of those but I don't know how to do that.

Reply:
If you wish, you can change the name of the library output by CMake using something like `set_target_properties($target PROPERTIES LIBRARY_OUTPUT_NAME $name)`

Reply:
Done.